chart/permissions-api/templates/job-migrate-database.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if has .Values.config.crdb.migrateHook (list "pre-sync" "manual") }}
```

Comment:
Do we anticipate needing or wanting to support these three kinds of migration workflows?

Reply:
manual is perhaps not needed. but pre-sync and init are useful.

pre-sync can't be used during the initial deployment as it will rely on configs and secrets that have to be synced first, and therefore won't be available. so you must use the init method. but after using the init method, you can switch to the pre-sync method which will ensure migrations only execute once per sync.

if having these options is not preferred we could get rid of it and just use init containers.